src/containerapp/azext_containerapp/custom.py

```diff
@@ -2863,7 +2863,10 @@ def create_session_pool(cmd,
                         target_port=None,
                         registry_server=None,
                         registry_pass=None,
-                        registry_user=None):
+                        registry_user=None,
+                        mi_user_assigned=None,
+                        registry_identity=None,
+                        mi_system_assigned=False):
```

Comment:
For `mi_system_assigned`, the default value could be `None`, which will be consistent with other commands.

```suggestion
                        mi_system_assigned=None):
```